Source/JavaScriptCore/runtime/Operations.h

```diff
@@ -895,15 +895,29 @@ ALWAYS_INLINE JSValue jsBitwiseXor(JSGlobalObject* globalObject, JSValue v1, JSV
 ALWAYS_INLINE EncodedJSValue getByValWithIndex(JSGlobalObject* globalObject, JSCell* base, uint32_t index)
 {
     if (base->isObject()) {
-        JSObject* object = asObject(base);
-        if (JSValue result = object->tryGetIndexQuickly(index))
+        if (JSValue result = asObject(base)->tryGetIndexQuickly(index))
```

Comment:
Could this just call getByValWithIndexAndThis with this = base?

Reply:
Yeah, changed.